lib/tapioca/rbi/printer.rb

```diff
@@ -118,7 +118,10 @@ class Comment
 
       sig { override.params(v: Printer).void }
       def accept_printer(v)
-        v.printl("# #{text.strip}")
+        text = self.text.strip
```

Comment:
Lol, I am so sure that you first wrote this as:
```ruby
text = text.strip
```
and you were wondering why it didn't work. This is such a gotcha part of the language.

Reply:
What do you mean?

Reply:
The fact that you have to use a `self.text` on the right hand side is so unintuitive to me. I, for one, have to keep thinking twice about that.